integration_tests/test_rough_align.py

```diff
@@ -417,8 +422,31 @@ def test_do_rough_alignment_with_scale(render, montage_scape_stack_with_scale, r
     assert(set(zvalues) == set(zs))
 
     yield output_lowres_stack
-    renderapi.stack.delete_stack(output_lowres_stack, render=render)
+    #renderapi.stack.delete_stack(output_lowres_stack, render=render)
+
+@pytest.fixture(scope='module')
+def test_do_rough_alignment_python(
+        render, montage_scape_stack, rough_point_match_collection,
+        tmpdir_factory, output_lowres_stack=None):
+
+    if output_lowres_stack == None:
+        output_lowres_stack = '{}_DS_Rough_python'.format(montage_scape_stack)
+    output_directory = str(tmpdir_factory.mktemp('output_json'))
+
+    solver_input = copy.deepcopy(test_rough_parameters_python)
```

Comment:
why do you need a deepcopy here?

Reply:
our parameters tend to be nested dictionaries. Things get messy with a shallow copy:
```
In [247]: import json

In [248]: print(json.dumps(a, indent=2))
{
  "b": {
    "c": 1
  }
}

In [249]: aa = dict(a)

In [250]: print(json.dumps(aa, indent=2))
{
  "b": {
    "c": 1
  }
}

In [251]: a['b']['c'] = np.pi

In [252]: print(json.dumps(aa, indent=2))
{
  "b": {
    "c": 3.141592653589793
  }
}

In [253]: xx = copy.deepcopy(a)

In [254]: print(json.dumps(xx, indent=2))
{
  "b": {
    "c": 3.141592653589793
  }
}

In [255]: a['b']['c'] = 1

In [256]: print(json.dumps(xx, indent=2))
{
  "b": {
    "c": 3.141592653589793
  }
}

In [257]: print(json.dumps(a, indent=2))
{
  "b": {
    "c": 1
  }
}
```